modules/widgets/src/info-widget.tsx

```diff
@@ -16,6 +16,10 @@ export type InfoWidgetProps = {
    * Additional CSS class.
    */
   className?: string;
+
+  mode: 'click' | 'hover' | 'static';
```

Comment:
Have you considered calling the click mode "sticky" instead?

Is "static" always visible? It's a little unclear what to expect based on the name alone

Reply:
Yea,

The idea with static is that the application decides to show (one or more) HTML overlays, independently of clicking and hovering. In the example, London could always have an info window for instance.

The app could add a number of InfoWidgets e.g. one for each row it wanted to highlight.